downstream/modules/automation-hub/proc-add-container-readme.adoc

```diff
@@ -25,10 +25,9 @@ Add a README to your container repository to provide instructions to your users
 
 .Procedure
 
-. Navigate to *Container Registry*.
-
+. Navigate to *Execution Environments*.
 . Select your container repository.
-. On the *Details* tab, click *Add*.
+. On the *Detail* tab, click *Add*.
 . In the *Raw Markdown* text field, enter your README text in Markdown.
 . Click *Save* when finished.
 
```

Comment:
The comment below step 5 does not work for me. After I save my text in Markdown, I only see the **Edit** button to edit and save again. 

Suggestion: `Once you add a README, you can edit it at any time by clicking **Edit** and repeating steps 4 and 5.`